packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.scss

```diff
@@ -0,0 +1,19 @@
+/* stylelint-disable max-nesting-depth */
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+:local {
+  .fill {
+    height: 100%;
+
+    :global {
+      .xfc {
+        height: 100%;
```

Comment:
`height: 100%` is not needed.

Reply:
By default the frame is dynamically resized to fit the content however in our use case we need to expand the frame to fit its parent. In order to achieve this I added an opt-in fill property that allowed the frame to expand to fit its parent. However, we could create a custom wrapper component that provides this functionality.